web/src/analytics_test_helpers.ts

```diff
@@ -1,12 +1,21 @@
-import fetchMock from "jest-fetch-mock"
+import fetchMock from "fetch-mock"
 import { Tags } from "./analytics"
 
+export function mockAnalyticsCalls() {
+  fetchMock.reset()
```

Comment:
if you have multiple mock helpers, it seems like it would be bad if they all called fetchMock.reset() when they set up?

Reply:
good poinit